src/main/java/org/openpnp/machine/reference/driver/TinygDriver.java

```diff
@@ -141,7 +145,7 @@ public void setEnabled(boolean enabled) throws Exception {
     @Override
     public void home(ReferenceHead head) throws Exception {
         // TODO: figure out how to home
-        // sendCommand("G28.2");
+        sendCommand("G28.2 X0 Y0");
```

Comment:
As @cri-s mentioned, this should home all axes. Should probably just be G28.2 with no axes specified.